macros/src/lib.rs

```diff
@@ -2,9 +2,27 @@
 
 extern crate proc_macro;
 
+use darling::FromMeta;
 use proc_macro::TokenStream;
 use quote::quote;
-use syn::{parse_macro_input, ItemFn};
+use syn::{parse_macro_input, AttributeArgs, ItemFn};
+
+// Handle arguments to our macros or default otherwise
+#[derive(Debug, FromMeta)]
+#[darling(default)]
+struct DeriveArgs {
```

Comment:
can we name this `TestArgs` maybe? since these are arguments passed to the test attribute.

Reply:
And to bench tho but ok.

Reply:
hmm yeah, be nice to not have #bench and #test, but anyway.

Reply:
calling attribute args, aka AttrArgs, as that's the real truth.